scripts/.babelrc.cjs

```diff
@@ -9,7 +9,7 @@ module.exports = {
         shippedProposals: true,
         useBuiltIns: 'usage',
         corejs: '3',
-        targets: { node: '18' },
+        targets: { node: '20' },
```

Comment:
Is this a breaking change for Node 18 users?

Reply:
No, this should only be used internally. The compilation target for public script is elsewhere.